apps/meteor/ee/server/models/raw/LivechatUnit.ts

```diff
@@ -190,4 +190,15 @@ export class LivechatUnitRaw extends BaseRaw<IOmnichannelBusinessUnit> implement
 	countUnits(): Promise<number> {
 		return this.col.countDocuments({ type: 'u' });
 	}
+
+	removeDepartmentFromUnit(unitId: string): Promise<UpdateResult> {
```

Comment:
Shouldn't we tie it to the BusinessUnit interface?
```suggestion
	removeDepartmentFromUnit(unitId: IOmnichannelBusinessUnit['_id']): Promise<UpdateResult> {
```